docs/getting-started-scalardl-ledger.md

```diff
@@ -0,0 +1,538 @@
+
+## Tools
+
+In this guide, we will use the following tools for testing.  
+
+1. Docker
+1. minikube
+1. kubectl
+1. Helm
+1. cfssl / cfssljson
+
+## Environment
+
+In this guide, we will create the following environment in your local by using Docker and minikube.  
+
+```
+On the Docker Network (named minikube)
+
+                   +-----------------------+
+  +-----------+    | +------------------+  |    +-----------+
+  |           |    | | Scalar DL Ledger |  |    |           |
+  |  Client   |    | +------------------+  |    | Cassandra |
+  | Container |    |                       |    | Container |
+  |           |    | Kubernetes (minikube) |    |           |
+  +-----+-----+    +-----------+-----------+    +-----+-----+
+        |                      |                      |
+*-------+----------------------+----------------------+-------*
+             ------->                     ------->
+               gRPC                    Cassandra Driver
+```
+
+
+## Step 1. Install tools
+
+First, you need to install the following tools that will be used in this guide.  
+
+1. Install the Docker according to the [Docker document](https://docs.docker.com/engine/install/)  
+
+1. Install the minikube according to the [minikube document](https://minikube.sigs.k8s.io/docs/start/)  
+
+1. Install the kubectl according to the [Kubernetes document](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)  
+
+1. Install the helm command according to the [Helm document](https://helm.sh/docs/intro/install/)  
+
+1. Install the cfssl and cfssljson according to the [CFSSL document](https://github.com/cloudflare/cfssl)
+
+## Step 2. Start minikube with docker driver
+
+In the after steps, we will create a Cassandra container as backend storage of the Scalar DL Ledger, on the same Docker Network as minikube. So, you need to start minikube with docker driver.  
+
+1. Start minikube with docker driver.
+   ```console
+   $ minikube start --driver=docker
+   ```
+
+1. Check minikube and the pods are running.
+   ```console
+   $ kubectl get pod -A
+   NAMESPACE     NAME                               READY   STATUS    RESTARTS      AGE
+   kube-system   coredns-64897985d-lbsfr            1/1     Running   1 (20h ago)   21h
+   kube-system   etcd-minikube                      1/1     Running   1 (20h ago)   21h
+   kube-system   kube-apiserver-minikube            1/1     Running   1 (20h ago)   21h
+   kube-system   kube-controller-manager-minikube   1/1     Running   1 (20h ago)   21h
+   kube-system   kube-proxy-gsl6j                   1/1     Running   1 (20h ago)   21h
+   kube-system   kube-scheduler-minikube            1/1     Running   1 (20h ago)   21h
+   kube-system   storage-provisioner                1/1     Running   2 (19s ago)   21h
+   ```
+   If the minikube started properly, you can see some pods in the kube-system namespace are `Running`.  
+
+## Step 3. Start Cassandra container
```

Comment:
Ditto
Similar steps exists in the Scalar DB document

Reply:
@kota2and3kan san,
Yes, It is easy to use.
So, you can confirm with @feeblefakie san.

Reply:
As I mentioned in another comment, I will keep this redundancy in this PR.